next/components/forms/segments/MunicipalServiceCard/MunicipalServiceCard.tsx

```diff
@@ -0,0 +1,135 @@
+    [Enum_Municipalservice_Icon.Greenery]: <TreeIcon className={className} />,
+    [Enum_Municipalservice_Icon.Lamp]: <LampIcon className={className} />,
+    [Enum_Municipalservice_Icon.SpatialPlanning]: <SpatialPlanningIcon className={className} />,
+    [Enum_Municipalservice_Icon.Waste]: <BasketIcon className={className} />,
+    [Enum_Municipalservice_Icon.Security]: <SecurityIcon className={className} />,
+    [Enum_Municipalservice_Icon.Marianum]: <MarianumIcon className={className} />,
+    [Enum_Municipalservice_Icon.Mosquito]: <MosquitoIcon className={className} />,
+    [Enum_Municipalservice_Icon.ChristmasTree]: <ChristmasTreeIcon className={className} />,
+    [Enum_Municipalservice_Icon.Housing]: <HousingIcon className={className} />,
+    [Enum_Municipalservice_Icon.Transport]: <TransportIcon className={className} />,
+    [Enum_Municipalservice_Icon.Excavations]: <ExcavationsIcon className={className} />,
+    [Enum_Municipalservice_Icon.ManagementCommunications]: (
+      <ManagmentCommunicationsIcon className={className} />
+    ),
+    [Enum_Municipalservice_Icon.Parking]: <ParkingIcon className={className} />,
+    [Enum_Municipalservice_Icon.Towing]: <TowIcon className={className} />,
+  } as const
+
+  return iconComponentMap[iconName]
+}
+
+const getTagStyle = (color: Enum_Municipalservice_Color) => {
+  const tagStyleMap: Record<Enum_Municipalservice_Color, string> = {
+    [Enum_Municipalservice_Color.Transport]: 'text-transport-700 bg-transport-100',
+    [Enum_Municipalservice_Color.Environment]: 'text-environment-700 bg-environment-100',
+    [Enum_Municipalservice_Color.Social]: 'text-social-700 bg-social-100',
+    [Enum_Municipalservice_Color.Education]: 'text-education-700 bg-education-100',
+    [Enum_Municipalservice_Color.Culture]: 'text-culture-700 bg-culture-100',
+    [Enum_Municipalservice_Color.Main]: 'text-main-700 bg-main-100',
+    [Enum_Municipalservice_Color.Marianum]:
+      'text-municipal-services-marianum bg-municipal-services-marianum-bg',
+    [Enum_Municipalservice_Color.Olo]: 'text-municipal-services-olo bg-municipal-services-olo-bg',
+    [Enum_Municipalservice_Color.Tsb]: 'text-municipal-services-tsb bg-municipal-services-tsb-bg',
+  } as const
+
+  return tagStyleMap[color]
+}
+
+type MunicipalServiceCardProps = {
+  service: MunicipalServiceFragment | MunicipalServiceHomepageFragment
+}
+
+const MunicipalServiceCard = ({ service }: MunicipalServiceCardProps) => {
+  if (!service.attributes) {
+    return null
+  }
+
+  return (
+    <ServiceCard
+      key={service.id}
+      title={service.attributes.title}
+      description={service.attributes.description}
+      buttonText={service.attributes.buttonText}
+      icon={getIconComponent(service.attributes.icon, service.attributes.color)}
+      href={service.attributes.href}
+      tags={service.attributes.tags?.data?.map((tag) => tag.attributes?.title ?? '') ?? []}
```

Comment:
I would go with this, to ensure you really have only proper `string[]` with no empty strings (as I said, we use `filter(isDefined)` a lot)
```ts
.map((tag) => tag.attributes?.title ?? '') ?? []
```
⬇️  
```ts
.map((tag) => tag.attributes?.title).filter(isDefined) ?? []
```